acrobat/scripts/scripts.js

```diff
@@ -290,6 +290,23 @@ const CONFIG = {
   imsScope: 'AdobeID,openid,gnav,pps.read,firefly_api,additional_info.roles,read_organizations',
 };
 
+function replaceDotMedia(area = document) {
+  // eslint-disable-next-line compat/compat
+  const currUrl = new URL(window.location);
+  const pathSeg = currUrl.pathname.split('/').length;
+  if (pathSeg >= 3) return;
```

Comment:
Why do we need this? I don't see it in CC's.

Reply:
They do have it on their code to run it just when is on root, here is the code:

https://github.com/adobecom/cc/blob/stage/creativecloud/scripts/utils.js#L61-L72